compiler-rt/lib/hwasan/hwasan_report.cpp

```diff
@@ -427,6 +436,9 @@ class BaseReport {
     bool is_allocated = false;
   };
 
+  void CopyShadow();
```

Comment:
Should we do the same as with `CopyAllocations`, i.e. return and assign?

Reply:
Isn't that here already https://github.com/llvm/llvm-project/commit/40ac72793accfdee3052567a5953bb3233e8539e ?

Reply:
done